lib/hammer_cli_foreman/trend.rb

```diff
@@ -0,0 +1,45 @@
+module HammerCLIForeman
+
+  class Trend < HammerCLIForeman::Command
+
+    resource :trends
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
```

Comment:
Since the name is optional for trend creation, I'd suggest to add here :label field as well.